lldb-eval/value.cc

```diff
@@ -285,72 +286,107 @@ void Value::Update(Value v) {
   }
 }
 
-Value CastScalarToBasicType(lldb::SBTarget target, Value val,
-                            lldb::SBType type) {
-  Value ret;
-
-  switch (type.GetCanonicalType().GetBasicType()) {
-#define CASE(basic_type, builtin_type)                   \
-  case basic_type: {                                     \
-    auto v = ConvertTo<builtin_type>(val.inner_value()); \
-    ret = CreateValueFromBytes(target, &v, type);        \
-    break;                                               \
+static llvm::APFloat CreateAPFloatFromAPSInt(const llvm::APSInt& value,
+                                             lldb::BasicType basic_type) {
+  switch (basic_type) {
+    case lldb::eBasicTypeFloat:
+      return llvm::APFloat(value.isSigned()
+                               ? llvm::APIntOps::RoundSignedAPIntToFloat(value)
+                               : llvm::APIntOps::RoundAPIntToFloat(value));
+    case lldb::eBasicTypeDouble:
+      // No way to get more precision at the moment.
```

Comment:
Did `clang-format` add this extra indentation? This way it may seem that the comment corresponds to the `lldb::eBasicTypeDouble` instead of `lldb::eBasicTypeLongDouble`. Would it make more sense to move it under `lldb::eBasicTypeLongDouble` or inline it with it?

Reply:
The extra indent is indeed from `clang-format`. I agree it looks a bit confusing, but other options seem even more confusing to me :)